src/main/java/org/jitsi/videobridge/EndpointMessageTransport.java

```diff
@@ -464,54 +574,128 @@ public void onStringData(WebRtcDataStream src, String msg)
     }
 
     /**
-     * Sends a specific <tt>String</tt> <tt>msg</tt> over the data channel of
-     * this <tt>Endpoint</tt>.
+     * Sends a specific message over one of the transport channels of this
+     * {@link EndpointMessageTransport}. Which channel to use is decided by
+     * the method based on the channel which was last active.
      *
      * @param msg message text to send.
      * @throws IOException
      */
-    public void sendMessageOnDataChannel(String msg)
+    void sendMessage(String msg)
         throws IOException
     {
         SctpConnection sctpConnection = endpoint.getSctpConnection();
+        ColibriWebSocket webSocket = this.webSocket;
         String endpointId = endpoint.getID();
 
-        if (sctpConnection == null)
+        Object dst = null;
+        if (webSocketLastActive)
```

Comment:
or maybe it's at least worth adding a helper to get the active connection so all that logic is tucked away somewhere else